tutorials/data-science/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Data Science on Google Cloud Platform
+description: Use Google Cloud Platform tools to automate, share, and scale Data Science workflows.
+author: jerjou
+tags: Data Science
+date_published: 2017-05-10
+---
+Data Science - a process for gleaning insight from data, and using said data to
```

Comment:
I suggest _that data_ instead of _said data_. Not huge deal, but it's more approachable.